### PR TITLE
Game over screen

### PR DESCRIPTION
**Pull request by tomasdivito**
Well, the game over screen is a thing now at least

### CODE REVIEW

File: app/gameover.rb
Comment (nsatragno) [inline app/gameover.rb]: nit: remove this comment.

File: app/gameover.rb
Comment (nsatragno) [inline app/gameover.rb]: nit: could you extract some of these magic numbers into constants?

File: app/player.rb
Comment (nsatragno) [inline app/player.rb]: nit: remove this comment.

**Review by nsatragno — COMMENTED** [review]
General comment:
I haven't downloaded and tested the patch but it looks like the game over screen pops up immediately after dying. Could we have a few frames where we display the death animation? Other than that, this LGTM.

File: app/gameover.rb
Comment (tomasdivito) [reply app/gameover.rb]: I think for some reason when i declare a variable with `ApplicationWindow::WINDOW_HEIGHT` i got an error but i will try

Comment (nsatragno) [reply app/gameover.rb]: I mean the 250, 60 and 1.0 that can be found on L21 and L19.

Comment (tomasdivito) [reply app/gameover.rb]: Oh that's ok

**Comment by tomasdivito** [comment]
I fixed the things that you marked on me but the part of the animation, i will introduce it in another patch.

**Review by nsatragno — APPROVED** [review]
Everything looks good, I'll merge this PR once #1 lands.

**Comment by tomasdivito** [comment]
Now you should be able to merge this with no problem :D

**Comment by tomasdivito** [comment]
Wait, no, it's not as it should... mmm

**Comment by nsatragno** [comment]
Looks like the merge got messed up. Ping me if you need help merging (also, it's "Lives", not "Lifes" :))

**Comment by nsatragno** [comment]
Merged as #7

**Pull request closed without merging**
